### PR TITLE
Create batch-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/batch-cohort.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
+2. In the Cohort section, click **Batch**.
+3. Click **Add another destination**.
+4. Enter **Name** and paste in the **API key** you copied from **Batch**.
+5. Map the Amplitude User ID field to the Batch User ID field
+6. Save when finished.
+
+## Send a cohort
+
+To sync your first cohort, follow these steps:
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:38]:
```suggestion
To sync your first cohort:
```

File: docs/data/destinations/batch-cohort.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. In Amplitude, open the cohort you want to sync, then click **Sync**.
+2. Select **Batch**, then click **Next**.
+3. Choose the account you want to sync to.
+4. Choose the sync cadence.
+5. When finished, save your work.
+
+## Locating your cohort in Batch
+
+Once the Cohort is synched to Batch, you should see it appear in the “Audiences” tab of the dashboard. You can then use it in a Push or In-App campaign by selecting it in the targeting section of the campaign editor. This is useful if you want to include or exclude a segment of users from your campaign.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:48]:
```suggestion
When the Cohort is syncs to Batch, it appears in the “Audiences” tab of the dashboard. Select it in the targeting section of the campaign editor to use it in a Push or In-App campaign. This is useful if you want to include or exclude a segment of users from your campaign.
```

File: docs/data/destinations/batch-cohort.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Use cases
+
+1. **Targeted Campaigns:** By syncing specific user cohorts from Amplitude, which segments users based on their behavior or attributes, to Batch, companies can create highly targeted push  or In-App campaigns. For example, a cohort of users who have abandoned their shopping carts could be targeted with a notification offering a discount to encourage completion of the purchase.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:52]:
```suggestion
- **Targeted Campaigns:** When you sync specific user cohorts from Amplitude to Batch, you can target those users more directly in push or In-App campaigns. For example, target a cohort of users who have abandoned their shopping carts with a notification that offers a discount to encourage completion of the purchase.
```

File: docs/data/destinations/batch-cohort.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+2. **User Retention:** Cohorts of users who have not engaged with the app or website for a set period can be identified in Amplitude and synced to Batch. These users can then be re-engaged with personalized content or updates relevant to their previous interactions, aiming to bring them back to the app or website.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:53]:
```suggestion
- **User Retention:** Identify cohorts of users who haven't engaged with the app or website for a set period in Amplitude and sync them to Batch. Re-engage these users with personalized content or updates relevant to their previous interactions, aiming to bring them back to the app or website.
```

File: docs/data/destinations/batch-cohort.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+3. **Product Feedback and Updates:** For users who have engaged with certain features of an app or service (a cohort identified in Amplitude), syncing this information to Batch would allow for sending specific notifications about updates, improvements, or requests for feedback on those features, ensuring that the most relevant users are being reached.
```
Comment (markzegarelli) [inline docs/data/destinations/batch-cohort.md:54]:
```suggestion
- **Product Feedback and Updates:** For users who engage with certain features of an app or service (a cohort identified in Amplitude), syncing this information to Batch allows for sending specific notifications about updates, improvements, or requests for feedback on those features, ensuring that you reach the most relevant users.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/destinations/batch-cohort.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+### Batch setup
+
+1. In Batch, navigate to Settings > General.
+2. Copy the API Key ("Live API Key" for mobile or "SDK API Key" for web) of the application you want to use Amplitude’s Cohorts in as well as your account's REST API Key.
+
+### Amplitude setup
+
+1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
+2. In the Cohort section, click **Batch**.
+3. Click **Add another destination**.
```
[the remark this answers is not in the document]
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/batch-cohort.md:31]: Screenshot 2023-11-22 at 1 01 44 AM

Yeah basically once you click on a destination tile, it brings up this modal "Add another destination" for those integrations without an overview side panel.

**Pull request merged**